aten/src/ATen/native/mkldnn/xpu/Linear.cpp

```diff
@@ -0,0 +1,95 @@
+#include <FusionUtils.h>
+#include <torch/library.h>
+
+namespace at::native::xpu {
+
+Tensor linear_pointwise(
+    const Tensor& input_t, // [M, K] or [B, M, K]
+    const Tensor& weight_t, // [N, K]
+    const c10::optional<Tensor>& bias_opt,
+    c10::string_view attr,
```

Comment:
c10::string_view is an alias of std::string_view and the community are planning to deprecate it. Please replace all in this PR with std::string_view.